WDAC-Policy-Wizard/app/src/AdvancedHunting.cs

```diff
@@ -152,13 +152,20 @@ private static string ConvertTimestampsToUTC(string record)
                     localTimeStamp = localTimeStamp.Replace(DEL_VALUE, ",");
 
                     // Parse the timestamp
-                    DateTime localDateTime = DateTime.ParseExact(localTimeStamp, timestampFormat, null);
+                    try
+                    {
+                        DateTime localDateTime = DateTime.ParseExact(localTimeStamp, timestampFormat, null);
 
-                    // Convert to UTC
-                    string utcDateTime = localDateTime.ToUniversalTime().ToString("o");
+                        // Convert to UTC
+                        string utcDateTime = localDateTime.ToUniversalTime().ToString("o");
 
-                    fields[0] = utcDateTime; 
-                    return string.Join(",", fields);
+                        fields[0] = utcDateTime;
+                        return string.Join(",", fields);
+                    }
+                    catch(Exception exp)
```

Comment:
perhaps not catch the generic `excception`?

Reply:
Not quite. [Exception is the highest level base class in C#](https://learn.microsoft.com/en-us/dotnet/api/system.exception?view=net-8.0) - any exception, whatsoever, will enter this catch statement when you might be only looking for a single, specific exception.